6_fetch_local_meteo.yml

```diff
@@ -0,0 +1,101 @@
+target_default: 6_fetch_local_meteo
+
+
+packages:
+  - dplyr
+  - tidyr
+  - scipiper
+  - feather
+  - readr
+
+file_extensions:
+  - feather
+  - ind
+
+sources:
+  - 6_drivers/src/local_meteo_utils.R
+  - 7_drivers_munge/src/GLM_driver_utils.R
+
+targets:
+  6_fetch_local_meteo:
+    depends:
+      - 7_drivers_munge/mendota_out/mendota_local_meteo.csv
+
+
```

Comment:
Are we likely to bring in local data for many more lakes?  I think the answer is no, in which case we might as well just merge this and let it coexist with the main many-lakes pipeline. But if the answer is yes, then `aoss` is cryptic and `airport` is quite generic, so I'd advocate for using the lakes IDs to name these functions and targets.

Reply:
I think the answer is no - I think this is paper-specific and not something we should use generally. I'm wondering if I should've PR'd this into a branch that stays solo, as it is related to getting things together for the paper, not necessarily for the project itself.

Reply:
Maybe a separate branch would be best, so this gets preserved somewhere we can find it again and where we're unlikely to accidentally delete it in next year's spring cleaning. But also, some messiness with multiple products from a project seems to be inevitable based on my experience to date, so maybe we just accept it.

Reply:
I am going to kill this an open up a PR onto a branch here.